### PR TITLE
Add screenshots to user guide (contacts)

### PR DESCRIPTION
**Pull request by ekweirui**
This pull request aims to do the following:
* Add screenshots for
  * `add-person`
  * `delete-person`
  * `delete-person-from-module`
  * `edit-person`

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -656,20 +673,21 @@ Format: `edit-person INDEX ([n/NAME] [e/EMAIL] [p/PHONE_NUMBER])`
...
-email `dinosaurlim@gmail.com`in Plannit.
+edit-person 2 n/Charmander e/charmander@gmail.com
 ```
-edit-person 1 n/Dinosaur Lim e/dinosaurlim@gmail.com p/91234567
-```
-In the above example, we are editing the contact with a displayed-index number of '1' on the screen to now have the name
-`Dinosaur Lim`, email `dinosaurlim@gmail.com`, and phone number `91234567` in Plannit.
+In the above example, we are editing the contact with a displayed-index number of '2' on the screen to now have the name
+`Charmander` and email `charmander@gmail.com`in Plannit.  
+Screenshots for the last example:
```
Comment (cheeheng) [inline docs/UserGuide.md:685]: The format here looks slightly different from what @teoyuqi wrote. Let's discuss.

**Review by cheeheng — COMMENTED** [review]
LGTM. Requires discussion on user guide format.

File: docs/UserGuide.md (hunk 1 above)
Comment (teoyuqi) [reply docs/UserGuide.md:685]: Perhaps a different phrasing is necessary as `edit-person` has more than one example?

Comment (ekweirui) [reply docs/UserGuide.md:698]: Good spot, I have changed the format accordingly.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -552,6 +552,10 @@ from the module with module code `CS2040`, using their corresponding link alias
 <br>
 
 ### 2.4. Contacts
+The terms "contact" and "person" has similar meanings and for the majority of the time, can be understood as referring
```
Comment (cheeheng) [inline docs/UserGuide.md:555]:
```suggestion
The terms "contact" and "person" have similar meanings and for the majority of the time, can be understood as referring
```
Grammatical error?

**Review by cheeheng — COMMENTED** [review]
LGTM except for a grammatical error

File: docs/UserGuide.md [hunk 3]
```diff
@@ -647,9 +664,15 @@ Format: `delete-person-from-module m/MODULE_CODE n/NAME`
...
+a screenshot of Plannit before and after executing the command:
+
+
+| Before executing the command                                                                                                                                              | After executing the command                                                                                                                                        |
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| {::nomarkdown}<p align="center"><img src="images/delete-person-from-module-before.png"/></p><p>Enter the command into the command box and hit <code>ENTER</code>.</p>{:/} | {::nomarkdown}<p align="center"><img src="images/delete-person-from-module-after.png"/></p><p>Charlotte Oliveiro has been deleted from the module CS2103T!</p>{:/} |
 
 <div markdown="span" class="alert alert-info">:information_source: **Note:**
```
Comment (teoyuqi) [inline docs/UserGuide.md:677]:
```suggestion
<div markdown="span" class="alert alert-info">:information_source: **Note:**<br>
```
Missing a `<br>`?

**Review by teoyuqi — COMMENTED** [review]
Just a minor nitpick

**Review by teoyuqi — APPROVED** [review]
LGTM

**Review by cheeheng — APPROVED** [review]
LGTM

**Pull request merged**